docs/usage/index.md

```diff
@@ -0,0 +1,78 @@
+# Using this package
+
+```{toctree}
+:maxdepth: 1
+:hidden: true
+
+package-structure
+code-examples
+```
+
+First of all, a Plone Distribution is a Python Package that can be installed by `pip` and follows certain conventions.
+
+## Create a new Plone Distribution
+
+The Plone Community provides a `Cookiecutter` template to generate new Plone Distribution packages: [cookiecutter-plone-distribution](https://github.com/collective/cookiecutter-plone-distribution).
+
+This template is active development and it will probably change its structure until a stable release of `plone.distribution` is made.
```

Comment:
```suggestion
This template is under active development.
It will probably change its structure until a stable release of `plone.distribution` is made.
```